test-clients/src/main/java/com/hedera/services/bdd/suites/file/FileUpdateSuite.java

```diff
@@ -98,7 +98,8 @@ private HapiApiSpec apiPermissionsChangeDynamically() {
 	}
 
 	private HapiApiSpec updateFeesCompatibleWithCreates() {
-		long origLifetime = 100_000_000;
+		final long origLifetime = 7_200_000L;
+		final long extension = 700_000L;
```

Comment:
Both `origLifetime` and `origLifetime + extension` will be in the default range of
https://github.com/hashgraph/hedera-services/blob/31d81b2df3e4f37762427d1671dd564160f5416b/hedera-node/src/main/resources/bootstrap.properties#L66-L67